### DISCUSSION TITLE
Could mv-action be executed with 'on mouse hover'?

### DISCUSSION THREAD

**Discussion by GalinhaLX** [mavoweb/mavo#890] [Q&A] [answered]
Hi

Is there a way to trigger mv-action when  hovering  an element ?

or maybe is there a way to execute a JavaScript, that would call mv-action, when moving the mouse pointer onto an element?

something like:
`<div onmouseover="mv-action()">hover me<div>`

**Comment by LeaVerou** [comment, marked as answer]
`<div onmouseover="this.click()">hover me<div>`

I would suggest using `onmouseenter` though.

**Comment by karger** [comment]
It seems unfortunate to have Mavo lose the information about which event occurred.   Consider e.g. if you want to handle two different events---you would have to use more complex methods (such as wrapping in an extra tag) to be able to tell mavo which event is being handled.

What if we introduced a general syntax of attributes like `mv-on-eventName` (`mv-on-click`, `mv-on-hover`, `mv-on-keypress`, etc.) to let us dispatch arbitrary events?
